.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       contents: read
     strategy:
       matrix:
-        node-version: [14, 16, 18, 20]
+        node-version: [18, 20, 21]
```

Comment:
Don't know if we should be testing on a non-LTS version? @fastify/plugins

Reply:
For me, -1.
But I see the others to agree on adding it.

Reply:
yes, we got multiple times regressions with non-LTS 21 in undici, which we could report upstream and get it fixed in timely manner.

Reply:
We currently test for 21 in /workflows

https://github.com/fastify/workflows/blob/39d0984518e137475ac519598cfe701509895321/.github/workflows/plugins-ci.yml#L104-L107

Reply:
The problem of supporting non-LTS line is that we cannot just remove it when EOL.
We need to proper deprecate the version and bump a major.
That is a disaster if it is about a hundred of package.

Reply:
One of the problem of `node@21` is that it is going to be EOL very soon (around 3 month later).
I imagine it would be EOL before we publish `fastify@5`.

![schedule](https://github.com/fastify/fastify-elasticsearch/assets/23028015/822fb1f6-45fb-4aed-8490-3c12f3f9ada1)

Reply:
I agree, I’ve seen interim releases being tested in workflows before and assumed this was the case

That only LTS lines were officially supported but CI would test against as many versions as possible for those who are interested and with no promises

@mcollina